onnxruntime/server/main.cc

```diff
@@ -108,5 +119,7 @@ int main(int argc, char* argv[]) {
       .NumThreads(config.num_http_threads)
       .Run();
 
+  grpc_app.Run();
```

Comment:
Make it configurable to turn on gRPC + HTTP. I'd rather just have one on by default.

Reply:
I'm ok to make both on by default, but would you mind check if the two ports are not the same? Users may make mistake here. :-)